app/src/main/res/drawable/search_bar_background.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2019 Derek Ellis. Subject to the MIT license.
+  -->
+
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+       android:shape="rectangle"
+       android:padding="10dp">
+
+    <stroke
+            android:width="1dp"
```

Comment:
Dimension?

Reply:
I don't think we need dimensions in a drawable file